core/src/test/java/org/frankframework/util/LockerTest.java

```diff
@@ -2,10 +2,10 @@
 
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.fail;
 import static org.junit.Assume.assumeFalse;
```

Comment:
Still using the JUnit4 `assumeFalse()` method and Hamcrest Matchers that are JUnit4 compatible.
However I'm aware by now that this is not a simple change.